libs/aws/odc/aws/queues.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+
+def get_queue(queue_name):
+    sqs = boto3.resource('sqs')
+    queue = sqs.get_queue_by_name(QueueName=queue_name)
+    return queue
+
+def publish_message(queue, msg):
+    resp = queue.send_message(
+        QueueUrl=queue.url,
+        MessageBody=msg
+    )
+    assert(resp['ResponseMetadata']['HTTPStatusCode'] == 200)
```

Comment:
Careful with parenthesis here. And this should probably be describing what happened and not just failing.
```suggestion
    assert resp['ResponseMetadata']['HTTPStatusCode'] == 200, "Failed to publish the message"
```

Reply:
Done.